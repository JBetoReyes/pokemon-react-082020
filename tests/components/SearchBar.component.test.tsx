import React from 'react';
import SearchBar from '@components/common/SearchBar.component';
import { shallow } from 'enzyme';

describe('SearchBarComponent', () => {
  let searchBar = shallow(<SearchBar title="title" />);

  beforeEach(() => {
    searchBar = shallow(<SearchBar title="title" />);
  });

  test('Should show the SearchBarComponent', () => {
    expect(searchBar).toMatchSnapshot();
  });
});
