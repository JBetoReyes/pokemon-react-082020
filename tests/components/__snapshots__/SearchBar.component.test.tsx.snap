// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBarComponent Should show the SearchBarComponent 1`] = `
<div
  className="search-bar__container"
>
  <h2
    className="search-bar__title"
  >
    title
  </h2>
  <form
    className="search-bar__form"
    onSubmit={[Function]}
  >
    <input
      className="search-bar__input"
      onChange={[Function]}
      type="text"
      value=""
    />
  </form>
</div>
`;
